SU-SWS/stanford_events_importer
# HSDO-593 Event recap fields

# READY FOR REVIEW

# Summary
- Added recap fields to show post event details.

# Needed By (Date)
- 5/10

# Steps to Test

1. Checkout branch, enable stanford_events_recap
2. create/edit a "Stanford Event"
3. Validate "Post Event Details" collapsed fieldset with 4 fields.
4. populate the fields and save the event.
5. Event Recap should appear at the top of the node page.
